tests/models/llama/test_tokenization_llama.py

```diff
@@ -510,7 +516,7 @@ def test_integration_test_xnli(self):
     def test_special_token_special_word(self):
         # the word inform should be split as ['in', 'form']
         tokenizer = LlamaTokenizer.from_pretrained("huggyllama/llama-7b", legacy=False)
-        tokenizer.add_tokens(["<REPR_END>"], special_tokens=True)
+        tokenizer.add_tokens(["<REPR_END>"], special_tokens=False)
```

Comment:
Because it needs to be an `added_token` to have spaces around it otherwise. I can see if we need this again or not but basically `spaces_between_special_tokens` would fail otherwise